tools/chef/Berksfile

```diff
@@ -25,6 +25,7 @@ cookbook 'jetty', :github => 'inviqa/chef-jetty', :tag => '0.2'
 cookbook 'solr', :github => 'inviqa/chef-solr', :tag => '1.0.1'
 cookbook 'chef-varnish', :github => 'inviqa/chef-varnish', :ref => '1.0.5'
 cookbook 'config-driven-helper', '~> 2.0'
+cookbook 'package-replace', :git => 'git@github.com:inviqa/chef-package-replace', :ref => 'master'
```

Comment:
any chance we can tag and release to supermarket? I'm on annual leave next week, but you have access to the inviqa Chef ID credentials